roles/kubespray-defaults/tasks/no_proxy.yml

```diff
@@ -22,7 +22,7 @@
       {%- if additional_no_proxy is defined -%}
       {{ additional_no_proxy }},
       {%- endif -%}
-      127.0.0.1,localhost,{{ kube_service_addresses }},{{ kube_pods_subnet }}
+      127.0.0.1,localhost,{{ kube_service_addresses }},{{ kube_pods_subnet }},svc,svc.{{ dns_domain }}
```

Comment:
note: this additional svc non_proxy configuration is for avoiding http proxy on k8s internal communication.